src/participants/trigger_service.py

```diff
@@ -233,7 +233,12 @@ def handle_wi(self, wid):
         new_service_xml = service % params
 
         # Replace the matching one:
-        services = etree.fromstring(services_xml)
+        try:
+            services = etree.fromstring(services_xml)
+        except etree.XMLSyntaxError as e:
```

Comment:
Instead of trusting that the XMLSyntaxError is caused by empty _service file, we should probably check if it's empty. It might be that the syntax error is caused by something else and in that case we don't wan't to just overwrite the _service file.